internal/proto/codec/builtin.go

```diff
@@ -1119,3 +1135,61 @@ func decodeLocalTime(buffer []byte, offset int32) (h, m, s, nanos int) {
 	nanos = int(FixSizedTypesCodec.DecodeInt(buffer, offset))
 	return h, m, s, nanos
 }
+
+func EncodeInt(buffer []byte, offset, value int32) {
+	binary.LittleEndian.PutUint32(buffer[offset:], uint32(value))
```

Comment:
TIL go does not have signed versions of these methods 😄

Reply:
:)